docs/deprecations.md

```diff
@@ -12,14 +12,6 @@ layout: Doc
 
 `maximumEventAge` and `maximumRetryAttempts` should be defined directly at function level. Support for those settings on `destinations` level, will be removed with v2.0.0
 
-<a name="AWS_HTTP_API_VERSION"><div>&nbsp;</div></a>
-
-## AWS HTTP API payload format
-
-Default HTTP API Payload version will be switched to 2.0 with next major release (For more details see [payload format documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html#http-api-develop-integrations-lambda.proxy-format))
-
-Configure `httpApi.payload` explicitly to ensure seamless migration.
-
```

Comment:
We should actually leave that. Some users will still use v1, and link to deprecation should remain working